appveyor.yml

```diff
@@ -10,28 +10,32 @@ environment:
     # Note: 'default' toolchain is the same as 'vs-14-2015'
 
     - TOOLCHAIN: "ninja-vs-12-2013-win64"
-      PROJECT_DIR: examples\foo
+      PROJECT_DIR: examples\libscrypt
 
     - TOOLCHAIN: "nmake-vs-12-2013-win64"
-      PROJECT_DIR: examples\foo
+      PROJECT_DIR: examples\libscrypt
 
     - TOOLCHAIN: "vs-12-2013-win64"
-      PROJECT_DIR: examples\foo
+      PROJECT_DIR: examples\libscrypt
 
     - TOOLCHAIN: "vs-12-2013-xp"
-      PROJECT_DIR: examples\foo
+      PROJECT_DIR: examples\libscrypt
 
     - TOOLCHAIN: "vs-12-2013"
-      PROJECT_DIR: examples\foo
+      PROJECT_DIR: examples\libscrypt
 
     - TOOLCHAIN: "vs-14-2015"
-      PROJECT_DIR: examples\foo
+      PROJECT_DIR: examples\libscrypt
+
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
```

Comment:
I guess `APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017` not needed for `vs-14-2015-win64`

Reply:
This actually does not seem to work. Maybe you need to set it everywhere.